app/views/layouts/application.html.haml

```diff
@@ -6,5 +6,58 @@
     = csrf_meta_tags
     = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    %script{:src => "https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"}
+    %script{:src => "http://maps.google.com/maps/api/js?sensor=tru", :type => "text/javascript"}
+    %script{:src => "https://cdnjs.cloudflare.com/ajax/libs/gmaps.js/0.4.25/gmaps.js"}
+
+    :css
+      .top-bar {background-color: #FFA500;}
+      .menu {background-color: #1F6C87;}
   %body
+
+    %nav.top-bar.nav-desktop
+      .wrap
+        .top-bar-left
+          %h3.site-logo Slow Food Online
+        .top-bar-right
+          %ul.menu.menu-desktop
+            %li
+              %a{:href => "#"} Registration
+            %li
+              %a{:href => "#"} Login
```

Comment:
Please use built in helpers for links. `link_to` (Please see the Rails documantation for how to use that syntax)